src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java

```diff
@@ -421,12 +443,10 @@ void doTaxSpecificRowSetup(ResultSet rs, PreparedStatement insertStatement, Stri
         @Override
         void doTaxSpecificSecondPassRowSetup(ResultSet rs, Transaction1042SSummary summary) throws SQLException {
             String financialObjectCode = rs.getString(summary.transactionDetailRow.finObjectCode.index);
-            String incomeClassCode;
+            String incomeClassCode = rs.getString(summary.transactionDetailRow.incomeClassCode.index);

```

Comment:
Now that the logic for determining the income class code has changed, it might be necessary to update the code for determining whether a transaction represents a fed-tax-withheld or state-tax-withheld amount. Lori's analysis of the results should reveal whether such a change is needed.